src/Attribute.h

```diff
@@ -0,0 +1,43 @@
+#ifndef ATTRIBUTE_H
+#define ATTRIBUTE_H
+
+#include <string>
+#include <vector>
+using namespace std;
+
+class AttributeGenerator;
+
+class Attribute
+{
+public:
+	string attribute;
+	int attribute_probability;
+	string attr_generate;
+	Attribute(string, int);
```

Comment:
Please declare also a parameter names, it will be more readible.

Reply:
Done.